### PR TITLE
[Konnected] Taking configured onValue into account when sending status to actuator

### PR DESCRIPTION
**Pull request by haavar**
Fixing a bug where the configured onValue for the channel was ignored when sending a status to the panel. The onValue configuration was only considered when receiving a status from the panel. Honoring the onValue configuration is important when using the standard relay boards recommended by Konnected that are low triggered.

### CODE REVIEW

File: bundles/org.openhab.binding.konnected/src/main/java/org/openhab/binding/konnected/internal/handler/KonnectedHandler.java
Comment (haavar) [inline bundles/org.openhab.binding.konnected/src/main/java/org/openhab/binding/konnected/internal/handler/KonnectedHandler.java:426]: The serialized payload is logged below, and provides more accurate information than the individual debug lines.

**Comment by jlaur** [comment]
> can you review this PR when you get a chance?

Sure.

> I have a few other changes queued, and I want to avoid large PRs and merge conflicts.

That's a good approach! And it will usually also result in faster reviews.

> Let me know if I can clarify anything.

Perhaps you could mention this **onValue** in the README as currently it seems undocumented? That would help me also understanding the logic behind it.

**Comment by haavar** [comment]
Thanks. I did not mean to nag, I just saw you approve my other (newer) PR, and wanted to see if there was any issues with this.

Added a paragraph about the onValue in the readme. The name for this parameter may not be the best. It's really for controlling whether whether the on value in openhab corresponds to high or low signal on the panel.

I used this setting when connecting a low level triggered relay board. That means that the relay turns on when the signal is set to 0V (ground / low). Since the onValue setting was not taken into account, I had to remember that for that zone ON was turning the relay OFF and vise versa. It got pretty confusing when writing rules using that zone. After fixing the onValue settings, the rules became much easier to understand.

File: bundles/org.openhab.binding.konnected/src/main/java/org/openhab/binding/konnected/internal/handler/KonnectedHandler.java
Comment (volfan6415) [inline bundles/org.openhab.binding.konnected/src/main/java/org/openhab/binding/konnected/internal/handler/KonnectedHandler.java:96]: any reason you are taking these logging statements out?  I get not wanting to clutter up the logs.  But i think maybe we keep them in but up the logger to logger.trace? 

Just so that they are still available for future debugging?

**Comment by jlaur** [comment]
@haavar - I will let @volfan6415 as code owner respond to the open comment/review the PR before merging.

File: bundles/org.openhab.binding.konnected/src/main/java/org/openhab/binding/konnected/internal/handler/KonnectedHandler.java
Comment (volfan6415) [reply bundles/org.openhab.binding.konnected/src/main/java/org/openhab/binding/konnected/internal/handler/KonnectedHandler.java:96]: ok that makes sense.

**Review by volfan6415 — APPROVED** [review]

**Review by jlaur — APPROVED** [review]
LGTM

**Pull request merged**
